client/blocks/eligibility-warnings/index.tsx

```diff
@@ -249,6 +249,7 @@ const mapDispatchToProps = {
 		saveSiteSettings( selectedSiteId, {
 			blog_public: 1,
 			wpcom_coming_soon: 0,
+			wpcom_public_coming_soon: 0,
```

Comment:
I don't think we should change the setting when the site goes atomic. We did this for v1 because coming soon would break things on atomic. But the idea with v2 is that sites aren't private to begin with, so don't need to be taken public.

Unless you're thinking this is just temporary until we've worked out syncing of options with atomic?

Reply:
Sounds good, will remove / add a comment on this.

Reply:
As discussed leaving this for now as I think it makes sense given the function name we need to figure out why / where this is being called if it is no longer needed.

Reply:
Tracked this down I think. The only time `makeSitePublic()` is called is if the list of AT holds includes `SITE_NOT_PUBLIC`. The list of holds usually comes from `/sites/:id/automated-transfers/eligibility`, however `SITE_NOT_PUBLIC` doesn't exist on the backend. I couldn't find it in the history either (although I didn't search exhaustively).

So under normal circumstances (like attempting to install a plugin or generate ftp credentials) the `<EligibilityWarnings>` UI doesn't call `makeSitePublic()` and set `wpcom_coming_soon` to `0`.

I did find something in Calypso that's reusing the `<EligibilityWarnings>` UI though, and it passes in a hard-coded `SITE_NOT_PUBLIC` prop in order to get the right copy and button behaviour.
If your site is _already atomic_ and you go to `/settings/performance/:siteSlug` in Calypso you'll see this message, which is just a `<EligibilityWarnings>` under the hood.
<img width="737" alt="Screenshot 2020-10-05 at 11 06 10 AM" src="https://user-images.githubusercontent.com/1500769/95028336-0926e000-06fc-11eb-831b-348a611ebfc1.png">

I believe this is because you can only set up Jetpack performance features if your site is public. Privacy prevents it from being able to cache your pages and images I suppose.

So I don't think we should be setting `wpcom_public_coming_soon` to `0` in this case. In this context `makeSitePublic()` really just means `makeSiteWorkWithJetpackPerformance()`, and it should work for any value of `wpcom_public_coming_soon`.

Reply:
Thanks this makes sense, I'll add a comment here in its place as I don't think this will be clear in code without it.

Reply:
> I don't think this will be clear in code without it
lol yup